### ISSUE TITLE
Flask-appconfig overwrites flasks binary

### ISSUE THREAD

**Issue by kszarlej** [mbr/flask-appconfig#5]
Currently flask==0.12 comes with a `flask` binary. flask's-appconfig `flask` utility overwrites it/gets overwritten by flasks original binary. 

IMO the utility should be renamed because i.e current flask docs gives instructions on how to use their original `flask` binary, and as it turned out I was trying to apply those instructions to a different binary :)

**Comment by mbr** [comment]
That's correct. The binary is actually obsolete, newer versions integrate with flask's own CLI interface. I'll look into this.
